kratos/containers/container_expression/specialized_container_expression_impl.h

```diff
@@ -54,11 +54,13 @@ void SpecializedContainerExpression<TContainerType, TContainerDataIO, TMeshType>
     const auto& r_container = this->GetContainer();
     const IndexType number_of_entities = r_container.size();
 
+    using raw_data_type = std::conditional_t<std::is_same_v<TDataType, int>, int, double>;
```

Comment:
um, what? Why? What about chars?

Reply:
We don't have variables of type `char`, thats why I was omitted the `char` in here. But, now did it anyways :)